native/android/src/main/java/com/nozbe/watermelondb/Database.kt

```diff
@@ -41,9 +44,35 @@ class Database(private val name: String, private val context: Context) {
 
     fun delete(query: SQL, args: QueryArgs) = db.execSQL(query, args)
 
-    fun rawQuery(query: SQL, args: RawQueryArgs = emptyArray()): Cursor = db.rawQuery(query, args)
+    fun rawQuery(sql: SQL, args: QueryArgs = emptyArray()): Cursor {
```

Comment:
@rozPierog Do you mind taking a look at this function? Maybe something stupid I did jumps at you...